charts/cloud-connector/README.tpl

```diff
@@ -14,92 +14,85 @@ $ pre-commit run -a
 
 # Cloud Connector
 
-[{{ .Project.Name }}]({{ .Project.URL }}) - {{ .Project.Description }}
+## Overview
 
-## TL;DR;
+This chart deploys the Sysdig Cloud Connector on your Kubernetes cluster, enabling Threat Detection and Image Scanning capabilities for the  AWS, GCP, and Azure Cloud providers.
 
-```
-$ helm repo add {{ .Repository.Name }} {{ .Repository.URL }}
-$ helm repo update
-$ helm upgrade --install {{ .Release.Name }} {{ .Repository.Name }}/{{ .Chart.Name }} \
-      --create-namespace -n {{ .Release.Namespace }}{{ with .Chart.Version }} --version={{.}} {{ end }} \
-      --set sysdig.secureAPIToken=SECURE_API_TOKEN
-```
-
-- [Configuration](#configuration)
-- [Configuration Detail](#configuration-detail)
-- [Usage examples](#usage-examples)
-- [Troubleshooting](#troubleshooting)
```

Comment:
index comes handy, advise not to remove it

Reply:
TOC is not used in a short single page. TOC serves for multi-pager installation.

Reply:
Install is already in the doc https://github.com/sysdiglabs/charts/tree/cloud-connector/charts/cloud-connector